bedrock/settings/static_media.py

```diff
@@ -1278,6 +1285,20 @@
         ),
         'output_filename': 'js/firefox_new_scene2-bundle.js',
     },
+    'experiment_firefox_new_scene1_all_links-pt2': {
```

Comment:
This should be an underscore before `pt2`? e.g. `experiment_firefox_new_scene1_all_links_pt2`